app/src/main/java/com/upa/passwordlocker/utils/CommonUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.upa.passwordlocker.utils;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+
+public class CommonUtils {
+
+    public static boolean hasWhiteSpace(String data) {
+
+        if(data == null) {
+            return false;
+        }
+
+        return data.contains(" ");
+    }
+
+    public static boolean hasActivityNotStarted(Context context, Intent intent) {
+        try {
+            context.startActivity(intent);
```

Comment:
ye hasActivityNotStarted method ko activity start krna chahiye kya ? Isko bola to nahi gya hai start krne. Isse pucha gaya hai ?